test/compiler/immutablearray.jl

```diff
@@ -70,14 +70,14 @@ function unescaped1_1(gen)
     a = [1,2,3,4,5]
     return gen(a)
 end
-let src = code_typed1(unescaped1, (Type{Core.ImmutableArray},))
+let src = code_typed1(unescaped1_1, (Type{Core.ImmutableArray},))
```

Comment:
is this really the best name?

Reply:
This or `no_escape_n` would be my approach but open to suggestions

Reply:
I'd like to go with `optimizable` and `unoptimizable`, and also propose to use `let` blocks to avoid unintended name leaks. c718fb99b0ec1fc7819837390a20a3cdf393bc75